src/wasm.h

```diff
@@ -86,6 +86,10 @@ struct Name : public cashew::IString {
   static Name fromInt(size_t i) {
     return cashew::IString(std::to_string(i).c_str(), false);
   }
+
+  bool hasSubstring(cashew::IString substring) {
```

Comment:
Well, I do like that pretty much all the core AST fits in one header. I think splitting out Name and Literal makes sense - they are more separate - but perhaps not Module, Const, Binary, Export, etc. - i.e. not module and its contents. I suppose that separation is subjective to some extent, though.